lib/vagrant-libvirt/action/create_networks.rb

```diff
@@ -63,7 +63,7 @@ def call(env)
 
               if @options[:ip]
                 handle_ip_option(env)
-              elsif @options[:type] == :dhcp
+              elsif @options[:type].to_s == 'dhcp'
```

Comment:
because `to_sym` method doesn't exist for NilClass, but `to_s` does.

so this code works in all cases, even for nil values.